xml/operating_system.xml

```diff
@@ -26,9 +26,9 @@
     <param pos="3" name="os.version"/>
   </fingerprint>
 
-  <fingerprint pattern="^(?i:(?:Microsoft )?Windows 10 Mobile(?:\s([a-z]+))?(?: Edition)?)$">
+  <fingerprint pattern="^(?i:(?:Microsoft )?Windows 10 Mobile(?:\s([a-z]+))??(?: Edition)?)$">
```

Comment:
To simplify reading this regex and to match the standard we were applying elsewhere I'd recommend moving the case insensitive modifier `(?i)` to the beginning before the carrot:

`(?i)^(?:Microsoft )?Windows 10 Mobile(?:\s([a-z]+))??(?: Edition)?$`

While we don't currently have a Python port of this library, it is important to know that the Python regex engine doesn't support the use of this modifier, `?i`,  in any location or construction other than the beginning of the line.

Prior efforts on this topic can be found here: https://github.com/rapid7/recog/pull/367